src/dmd/backend/gloop.c

```diff
@@ -1268,6 +1268,8 @@ STATIC void markinvar(elem *n,vec_t rd)
  *      e = defnod[] entry that is an assignment to a variable
  */
 
+extern "C"
+{
 void fillInDNunambig(vec_t v, elem *e)
```

Comment:
Isn't this change unrelated?

Reply:
No. I set it to C to avoid name mangling issues with `vec_t` which is our annoying friend `size_t`.